regexec.c

```diff
@@ -11154,7 +11154,7 @@ S_reginclass(pTHX_ regexp * const prog, const regnode * const n, const U8* const
 }
 
 STATIC U8 *
-S_reghop3(U8 *s, SSize_t off, const U8* lim)
+S_reghop3(pTHX_ U8 *s, SSize_t off, const U8* lim)
```

Comment:
Shouldn't this be part of a later commit?

Reply:
Probably. Now if only github would show me what commit they ended up in, I could easily tell…